client/containers/Users/Dialogs/RemoveMultiFactorDialog.jsx

```diff
@@ -47,6 +55,21 @@ export default connectContainer(class extends Component {
     const message = getDialogMessage(messageFormat, 'username',
       getName(user, userFields, languageDictionary));
 
+    const fields = _.cloneDeep(userFields) || [];
+    const providers = user && user.multifactor ? user.multifactor : [];
```

Comment:
could be simplified to just `(user && user.multifactor) || []`. parens are really unnecessary, but make it clearer for the next person.
change isn't required- just throwing it out there.